README.md

```diff
@@ -21,17 +21,20 @@ This extension splits Jupyter edit mode into two modes: Vim command mode and Vim
 Three editing modes now exist: Jupyter command, Vim command, and Vim insert.
 
 ## Install
-### Prerequisites
+### For jupyterlab 3+
 
-* JupyterLab 2.0
+```bash
+pip install ????????????
```

Comment:
I don't know what name this will end up with on pypi so ??? for now.

Reply:
Maybe `jupyterlab-vim` ? `-` seems to be more used than `_` for package name in pip.

Reply:
I like consistency everywhere. Also if you name the python package with an `_` than you still pip install using `-`, I guess pip just substitutes them.

Reply:
I didn't know about the substitution, then it might be easier to go with `jupyterlab_vim`.

Reply:
Done.